src/components/NurseryWithdrawals/ReassignmentRenderer.tsx

```diff
@@ -97,7 +97,7 @@ export default function ReassignmentRenderer({ plots, setReassignment }: Reassig
             id={`quantity_${plantingId}`}
             type='number'
             min={0}
-            onChange={onUpdateQuantity}
+            onChange={(newValue) => onUpdateQuantity(`quantity_${plantingId}`, newValue)}
```

Comment:
we can remove the id here, it is unused,, also maybe remove it from onUpdateQuantity